### ISSUE TITLE
Include option to display title on charts

### ISSUE THREAD

**Issue by tomkinsc** [marbl/Krona#17]
Along with the other [chart display options](https://github.com/marbl/Krona/wiki/Changing%20chart%20viewing%20defaults), it would be helpful to be able to pass in a chart title to be displayed above the plot.

**Comment by ondovb** [comment]
Is this something that could be inherent to each chart (done at import), or would you want control over each view of it via GET variables (or both)? Currently, you can name the topmost level, which appears in the center when zoomed out, with `-n` or sample names, but I agree that an additional chart title would be useful.

**Comment by tomkinsc** [comment]
Thanks for the quick attention! Having the title inherent to the chart would be fine for us, though others may want the flexibility to change it with GET params. For taxonomic hierarchies, it's still helpful to have 'root' in the center of the the plot, and it would be nice to have the ability to (for example) include a sample name above the chart. When we're flipping through multiple Krona reports, including a title would make it easy to see which chart is which.

I actually didn't know about the `-n` option. Is that mentioned in the docs somewhere?

**Comment by ondovb** [comment]
It's in the usage information for the commands (i.e. when run with no input; most import commands have the -n option). There is also a way to name datasets in a chart (after the filename with a comma), although the names only appear when there is more one, and then only in a dropdown and briefly in the center when switching. It may make sense to simply expand this feature so the name always appears at the top when provided.

**Comment by Carambakaracho** [comment]
Just for the record, I'd second @tomkinsc's wish to define chart titles (in case a second opinion would make a difference) Thanks

**Comment by ondovb** [comment]
Yep, knowing about interest in features is always helpful, thanks!

**Comment by stevekm** [comment]
I was looking for this feature as well. Having a formal chart title available for each chart would definitely be helpful; right now I am generating a chart for each dataset individually, then generating snapshots of the rendered page automatically. In this case, the default names do not appear. I am trying out the `-n` option instead. Thanks.

**Comment by kubu4** [comment]
I'd also like to add a request for this feature. I'd love it if the `name` of the currently selected chart would be displayed - similar to when the name is _very briefly_ displayed when switching charts (see example below). I'd love it if it could be permanently added. 

Alternatively, specifying a root name for each input file would be another option.

Thanks for an already great set of tools!

---

![Krona Tools screencap with dataset name briefly displayed when switching between charts](https://user-images.githubusercontent.com/4514104/59368541-55723380-8cf3-11e9-9bf6-5f6d314042d9.png)
